clerk/clerk.go

```diff
@@ -172,10 +173,21 @@ func FetchJwt(ctx context.Context) (string, error) { //nolint:funlen,cyclop
 		return "", fmt.Errorf("error creating request: %w", err)
 	}
 
-	for k, v := range clerkLogin.Cookies {
+	for cookieName, cookieValue := range clerkLogin.Cookies {
```

Comment:
Could you please add a reference to where `net/http` does this? Also, seeing some linter warnings that you'll need to fix.

Reply:
Yeah can do.

Re: the linter errors, I have another PR open where those get fixed: https://github.com/amp-labs/cli/pull/64